### ISSUE TITLE
PyPI tarball - some test data missing

### ISSUE THREAD

**Issue by frenzymadness** [maxmind/MaxMind-DB-Reader-python#60]
I'd like to update RPM package in Fedora but the tests fail because some test files in the tarball from PyPI are missing.

Should I download the test data from the git submodule or you can fix this? I think that it might be a good idea to include all or nothing so I guess that this is a bug.
```
======================================================================
ERROR: test_database_with_invalid_utf8_key (reader_test.TestAutoReader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/maxminddb-1.5.3/tests/reader_test.py", line 291, in test_database_with_invalid_utf8_key
    reader = open_database(
  File "/builddir/build/BUILD/maxminddb-1.5.3/maxminddb/__init__.py", line 44, in open_database
    return maxminddb.extension.Reader(database)
FileNotFoundError: No such file or directory: 'tests/data/bad-data/maxminddb-python/bad-unicode-in-map-key.mmdb'

======================================================================
ERROR: test_database_with_invalid_utf8_key (reader_test.TestExtensionReader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/maxminddb-1.5.3/tests/reader_test.py", line 291, in test_database_with_invalid_utf8_key
    reader = open_database(
  File "/builddir/build/BUILD/maxminddb-1.5.3/maxminddb/__init__.py", line 44, in open_database
    return maxminddb.extension.Reader(database)
FileNotFoundError: No such file or directory: 'tests/data/bad-data/maxminddb-python/bad-unicode-in-map-key.mmdb'

======================================================================
ERROR: test_database_with_invalid_utf8_key (reader_test.TestExtensionReaderWithIPObjects)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/maxminddb-1.5.3/tests/reader_test.py", line 291, in test_database_with_invalid_utf8_key
    reader = open_database(
  File "/builddir/build/BUILD/maxminddb-1.5.3/maxminddb/__init__.py", line 44, in open_database
    return maxminddb.extension.Reader(database)
FileNotFoundError: No such file or directory: 'tests/data/bad-data/maxminddb-python/bad-unicode-in-map-key.mmdb'

======================================================================
ERROR: test_database_with_invalid_utf8_key (reader_test.TestFDReader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/maxminddb-1.5.3/tests/reader_test.py", line 291, in test_database_with_invalid_utf8_key
    reader = open_database(
  File "/usr/lib/python3.8/site-packages/mock/mock.py", line 1092, in __call__
    return _mock_self._mock_call(*args, **kwargs)
  File "/usr/lib/python3.8/site-packages/mock/mock.py", line 1149, in _mock_call
    result = effect(*args, **kwargs)
  File "/builddir/build/BUILD/maxminddb-1.5.3/tests/reader_test.py", line 45, in get_reader_from_file_descriptor
    with open(filepath, "rb") as mmdb_fh:
FileNotFoundError: [Errno 2] No such file or directory: 'tests/data/bad-data/maxminddb-python/bad-unicode-in-map-key.mmdb'

======================================================================
ERROR: test_database_with_invalid_utf8_key (reader_test.TestFileReader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/maxminddb-1.5.3/tests/reader_test.py", line 291, in test_database_with_invalid_utf8_key
    reader = open_database(
  File "/builddir/build/BUILD/maxminddb-1.5.3/maxminddb/__init__.py", line 46, in open_database
    return maxminddb.reader.Reader(database, mode)
  File "/builddir/build/BUILD/maxminddb-1.5.3/maxminddb/reader.py", line 56, in __init__
    self._buffer = FileBuffer(database)
  File "/builddir/build/BUILD/maxminddb-1.5.3/maxminddb/file.py", line 16, in __init__
    self._handle = open(database, "rb")
FileNotFoundError: [Errno 2] No such file or directory: 'tests/data/bad-data/maxminddb-python/bad-unicode-in-map-key.mmdb'

======================================================================
ERROR: test_database_with_invalid_utf8_key (reader_test.TestMMAPReader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/maxminddb-1.5.3/tests/reader_test.py", line 291, in test_database_with_invalid_utf8_key
    reader = open_database(
  File "/builddir/build/BUILD/maxminddb-1.5.3/maxminddb/__init__.py", line 46, in open_database
    return maxminddb.reader.Reader(database, mode)
  File "/builddir/build/BUILD/maxminddb-1.5.3/maxminddb/reader.py", line 51, in __init__
    with open(database, "rb") as db_file:
FileNotFoundError: [Errno 2] No such file or directory: 'tests/data/bad-data/maxminddb-python/bad-unicode-in-map-key.mmdb'

======================================================================
ERROR: test_database_with_invalid_utf8_key (reader_test.TestMMAPReaderWithIPObjects)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/maxminddb-1.5.3/tests/reader_test.py", line 291, in test_database_with_invalid_utf8_key
    reader = open_database(
  File "/builddir/build/BUILD/maxminddb-1.5.3/maxminddb/__init__.py", line 46, in open_database
    return maxminddb.reader.Reader(database, mode)
  File "/builddir/build/BUILD/maxminddb-1.5.3/maxminddb/reader.py", line 51, in __init__
    with open(database, "rb") as db_file:
FileNotFoundError: [Errno 2] No such file or directory: 'tests/data/bad-data/maxminddb-python/bad-unicode-in-map-key.mmdb'

======================================================================
ERROR: test_database_with_invalid_utf8_key (reader_test.TestMemoryReader)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/maxminddb-1.5.3/tests/reader_test.py", line 291, in test_database_with_invalid_utf8_key
    reader = open_database(
  File "/builddir/build/BUILD/maxminddb-1.5.3/maxminddb/__init__.py", line 46, in open_database
    return maxminddb.reader.Reader(database, mode)
  File "/builddir/build/BUILD/maxminddb-1.5.3/maxminddb/reader.py", line 60, in __init__
    with open(database, "rb") as db_file:
FileNotFoundError: [Errno 2] No such file or directory: 'tests/data/bad-data/maxminddb-python/bad-unicode-in-map-key.mmdb'
```

**Comment by oschwald** [comment]
I'll fix this. Thanks for reporting the issue.

**Comment by oschwald** [comment]
This should be fixed in 1.5.4.

**Comment by frenzymadness** [comment]
It is, thank you!
